test/arrays/bugs/innerArrComp.chpl

```diff
@@ -0,0 +1 @@
+var comp = [[1,2,3] == [1,2]];
```

Comment:
Note that your two arrays being compared with `==` need to be the same size for this to be legal.

I'd probably suggest doing a `writeln(comp)` after this line to lock in that it's not just compiling, but also generating the expected result (which I think would be something like `true, true, true` once they're the same size?)

Reply:
Alright, I will make sure to add that in to this PR, thanks